src/auth/oidc.ts

```diff
@@ -38,6 +40,14 @@ export async function initOidc(authConf: CartaOidcAuthConfig) {
     console.log(`Setting up JWKS management for ${idpConfig.data['jwks_uri']}`);
     jwksManager = jose.createRemoteJWKSet(new URL(idpConfig.data['jwks_uri']));
 
+    // Set logout redirect URL
+    if (authConf.postLogoutRedirect !== undefined) {
+        postLogoutRedirect = authConf.postLogoutRedirect;
+    }
+    else {
+        postLogoutRedirect = `${ServerConfig.serverAddress}`;
```

Comment:
why can't we just use `postLogoutRedirect = ServerConfig.serverAddress?`

Reply:
`postLogoutRedirect` is a `string`; `ServerConfig.serverAddress` is `string | undefined`

Could be rewritten as something like `postLogoutRedirect = ServerConfig.serverAddress ?? ''` if I understand [this](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing) correctly?

Could also update `postLogoutRedirect` to `string | undefined` and then instead of redirecting to dashboard or specified URL just leave the user at the identity provider's logout page?  (Potentially replace the if/else with ternary operator).

Reply:
`postLogoutRedirect = ServerConfig.serverAddress ?? ''` that's better, otherwise we can change the postLogoutRedirect type. At the moment, you'll end up with `postLogoutRedirect` as `"undefined"`, not `undefined`.

Reply:
Think that the latest change should resolve this.